R/analysis_generic.R

```diff
@@ -0,0 +1,86 @@
+# # Inputting the data in long form
+# Inputting the data in long form
+bugsnetdata <- function(data, metaoutcome, treatment_list){
```

Comment:
Would it be possible to add a line just very briefly saying what `bugsnetdata` does?